src/features/Home/TipsModal/index.tsx

```
@@ -77,69 +77,92 @@ const TipsModal = ({
+                    return (
                       <div
-                        className={s.Category_Title}
-                        style={{
-                          backgroundColor: 'rgba(255, 255, 255, 0.8)',
-                          color: item?.categoryColor,
-                        }}
+                        className={s.Category_Content}
+                        style={{ backgroundColor: item?.categoryColor }}
                       >
-                        {item.title}
+                        <div className={s.Category_Color} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)